dts/bindings/base/base.yaml

```diff
@@ -1,6 +1,6 @@
 # Common fields for all devices
 
-include: [pm.yaml]
+include: [pm.yaml, "zephyr,device.yaml"]
```

Comment:
One problem with the approach of putting this in base.yaml is that it will affect every binding, even those bindings which do not correspond to struct devices.

Reply:
I am little bit confused here, how this file may be used by bindings that will not generate any struct device instance? Seems counter intuitive, relatively to the comment telling "common fields for all devices".   Do you have any example ?
PM stuff are fully tightened to struct device also.

Or should we have an intermediate device.yaml file that would include base, pm and the one I am adding then? And actual bindings generating struct device would include that one instead of base only.